docs/deploy_logging.md

```diff
@@ -0,0 +1,49 @@
+# Deploy openshift-logging on cluster
+
+### Pre-requisites before starting to deploy openshift-logging
+
+###### It is recommended to have minimum m5.xlarge setup.
```

Comment:
You shouldn't skip level of Sub-heading from ### to ###### but just increase one to ####